### PR TITLE
Vault of magic fixes

### PR DESCRIPTION
**Pull request by mshea**
Checking Vault of Magic items for formatting issues and missing data. Got through the Es so far.

### CODE REVIEW

File: data/vault_of_magic/magicitems.json [hunk 1]
```diff
@@ -4108,7 +3920,8 @@
   },
   {
     "name": "Ring of the Jarl",
-    "desc": "This thick band of hammered yellow gold is warm to the touch even in the coldest of climes. While you wear it, you have resistance to cold damage. If you are also wearing _boots of the winterlands_, you are immune to cold damage instead. These rings are highly valued magic items among the warriors of the Northlands. Chieftains and reaver leaders eagerly hunt for these rings, as they are a mark of prestige. Rings created by the _ring of the jarl_ are also prized, and owners of these magic rings gift the duplicates to their most worthy thegns or those who do them special service. It is rumored that some of these rings are cursed. The ill-fated wearers of such rings find themselves overcome with greed and able to produce rings without giving them away, creating a hoard the wearers cannot bear to part with or leave unguarded.",
+    "desc": "This thick band of hammered yellow gold is warm
```
Comment (dprothero) [inline data/vault_of_magic/magicitems.json:3923]: errant line break here @mshea

Comment (mshea) [reply data/vault_of_magic/magicitems.json]: Fixed and resubmitted. Thanks!

File: data/vault_of_magic/magicitems.json
Comment (augustjohnson) [inline data/vault_of_magic/magicitems.json:1]: There are a bunch of _spell names_ surrounded by underscores.  That's probably worth fixing before PR.

Comment (mshea) [reply data/vault_of_magic/magicitems.json:1]: Resubmitted with underscores around spell names removed. I did a global search, looked for any anomalies, and ensured it didn't screw up "page_no". It looked good on my sampling.

Comment (augustjohnson) [reply data/vault_of_magic/magicitems.json:1]: Awesome.

**Review by augustjohnson — APPROVED** [review]

**Pull request merged**